pkg/ccl/backupccl/backup_destination.go

```diff
@@ -94,19 +95,27 @@ func resolveDest(
 	var chosenSuffix string
 	var err error
 
+	// TODO(MB): future refactor just call getURSIByLocalityKV here from to and ""
```

Comment:
I'm not sure I understand this TODO -- are you suggesting we remove `defaultURI` as an argument?

Reply:
Yea, which is exactly what's described in Paul's ToDo above the function signature. I'll delete this.